subsys/net/lib/nrf_cloud/src/nrf_cloud_transport.c

```diff
@@ -861,10 +821,8 @@ int nct_dc_connect(void)
 	LOG_DBG("nct_dc_connect");
 
 	struct mqtt_topic subscribe_topic = {
-		.topic = {
-			.utf8 = nct.dc_rx_endp.utf8,
-			.size = nct.dc_rx_endp.size
-		},
+		.topic = { .utf8 = nct.dc_rx_endp.utf8,
```

Comment:
I'm not sure why clang-format does this, but such a brace placement violates the coding standard we follow:
https://www.kernel.org/doc/html/latest/process/coding-style.html#placing-braces-and-spaces
It seems to go wild after encountering nested designated initializers.

Reply:
An interesting observation, if the last item on the struct initializer lists ends with a comma, the whole expression is formated correctly, don't ask me why:
```
struct mqtt_topic subscribe_topic = {
	.topic = {
		.utf8 = nct.dc_rx_endp.utf8,
		.size = nct.dc_rx_endp.size,
	},
	.qos = MQTT_QOS_1_AT_LEAST_ONCE,
};
```
Otherwise:
```
struct mqtt_topic subscribe_topic = {
	.topic = { .utf8 = nct.dc_rx_endp.utf8,
		   .size = nct.dc_rx_endp.size },
	.qos = MQTT_QOS_1_AT_LEAST_ONCE
};
```

Reply:
reverted change.